layouts/default.vue

```diff
@@ -99,7 +106,12 @@
     },
 
     watch: {
-      '$i18n.locale': '$fetch'
+      '$i18n.locale': '$fetch',
+      $route() {
+        this.$nextTick(() => {
+          this.$refs.toppage.focus();
```

Comment:
Shouldn't we be reusing the existing `#main` element here?

Reply:
After a route change we should reset the focus to the container div so that the skip link is the first focused on element after a tab. That's how I wrote it now. There are also arguments to put it directly on the skip link, but in that case the skip link would be visible after each route change and that's undesirable for our visual users.

Reply:
In the end Mirjam and I concluded there is not an ideal solution. We decided that adding an empty element right before the skip link would work best. Setting focus on a wrapper causes unwanted behavior for screen readers (read to much).